spec/assisted_workflow/addons/git_spec.rb

```diff
@@ -54,33 +54,45 @@
     
     before do
       mock(@git).system("git status --porcelain"){ "" }
-      mock(@git).system("git checkout flavio.00001.new_feature")
+      mock(@git).system("git checkout flavio.1234.new_feature")
       mock(@git).system("git checkout master")
       mock(@git).system("git pull --rebase")
     end
     
     it "returns true if current branch is merged into master" do
-      mock(@git).system("git branch --merged"){ "flavio.00001.new_feature" }
+      mock(@git).system("git branch --merged"){ "flavio.1234.new_feature" }
       @git.check_merged!.must_equal true
     end
     
     it "returns false if current branch is not merged into master" do
-      mock(@git).system("git branch --merged"){ "flavio.00002.other_feature" }
+      mock(@git).system("git branch --merged"){ "flavio.1234.other_feature" }
       proc { @git.check_merged! }.must_raise AssistedWorkflow::Error, "this branch is not merged into master"
       
     end
   end
   
   it "removes current branch and its remote version" do
-    mock(@git).system("git push origin :flavio.00001.new_feature")
+    mock(@git).system("git push origin :flavio.1234.new_feature")
     mock(@git).system("git checkout master")
-    mock(@git).system("git branch -D flavio.00001.new_feature")
+    mock(@git).system("git branch -D flavio.1234.new_feature")
     @git.remove_branch
   end
   
   private #==================================================================
   
   def story
-    @story ||= PivotalTracker::Story.new(:id => "00001", :name => "New Feature", :description => "Feature description", :other_id => "flavio")
+    # stubs
+    @client = TrackerApi::Client.new(token: "mypivotaltoken")
+    stub(TrackerApi::Client).new{ @client }
+
+     any_instance_of(TrackerApi::Resources::Story) do |klass|
+      stub(klass).comments { [] }
+      stub(klass).tasks { [] }
+    end
```

Comment:
`end` at 91, 4 is not aligned with `any_instance_of(TrackerApi::Resources::Story) do |klass|` at 88, 5